tests/integration/ha_tests/test_ha.py

```diff
@@ -19,24 +19,31 @@
     clear_db_writes,
     count_primaries,
     count_writes,
+    cut_network_from_unit,
```

Comment:
Lmao at this point I'd just import `helpers` and call `helpers.<method>`. 

Definitely don't need to do so in this PR though.

Reply:
🤣